File: src/main/java/mods/orca/mffs/proxy/ClientProxy.java
package mods.orca.mffs.proxy;

import mods.orca.mffs.MFFSMod;
import net.minecraft.client.renderer.block.model.ModelResourceLocation;
import net.minecraft.item.Item;
import net.minecraftforge.client.model.ModelLoader;

public class ClientProxy extends Proxy {
    @Override
    public void registerItemRenderer(Item item, int metadata, String id) {
        ModelLoader.setCustomModelResourceLocation(item, metadata, new ModelResourceLocation(
                item.getRegistryName(), id
        ));
    }
}

File: src/main/java/mods/orca/mffs/proxy/ClientProxy.kt
package mods.orca.mffs.proxy

import net.minecraft.client.renderer.block.model.ModelResourceLocation
import net.minecraft.item.Item
import net.minecraftforge.client.model.ModelLoader

class ClientProxy : Proxy() {

    override fun registerItemRenderer(item: Item, metadata: Int, id: String) {
        ModelLoader.setCustomModelResourceLocation(
            item,
            metadata,
            ModelResourceLocation(
                requireNotNull(item.registryName),
                id
            )
        )
    }

}
